src/node/hooks/i18n.js

```diff
@@ -101,7 +101,7 @@ exports.expressCreateServer = function(n, args) {
   var localeIndex = generateLocaleIndex(locales);
   exports.availableLangs = getAvailableLangs(locales);
 
-  args.app.get ('/locales/:locale', function(req, res) {
+  args.app.get ('/static/locales/:locale', function(req, res) {
```

Comment:
does this change any proxy rewrite rules at all?

Reply:
No need proxy rules anymore

Reply:
https://github.com/ether/etherpad-lite/wiki/How-to-put-Etherpad-Lite-behind-a-reverse-Proxy -- have you tested with each of these configs?  Do they all work?

Reply:
It works perfectly 💯